modules/host.go

```diff
@@ -111,14 +119,33 @@ type (
 	// StorageObligation contains information about a storage obligation that
 	// the host has accepted.
 	StorageObligation struct {
+		ObligationId             types.FileContractID `json:"obligationid"`
+		FileSize                 uint64               `json:"filesize"`
+		SectorRootsCount         uint64               `json:"sectorrootscount"`
+		ContractCost             types.Currency       `json:"contractcost"`
+		LockedCollateral         types.Currency       `json:"lockedcollateral"`
+		PotentialDownloadRevenue types.Currency       `json:"potentialdownloadrevenue"`
+		PotentialStorageRevenue  types.Currency       `json:"potentialstoragerevenue"`
+		PotentialUploadRevenue   types.Currency       `json:"potentialuploadrevenue"`
+		RiskedCollateral         types.Currency       `json:"riskedcollateral"`
+		TransactionFeesAdded     types.Currency       `json:"transactionfeesadded"`
```

Comment:
fields within a block should be sorted. A few of the fields in this block are not sorted